lib/modules/apostrophe-schemas/index.js

```diff
@@ -1402,6 +1409,15 @@ module.exports = {
       converters: {
         string: function(req, data, name, object, field, callback) {
           object[name] = self.apos.launder.integer(data[name], field.def, field.min, field.max);
+          if (field.required && !data[name]) {
```

Comment:
This would reject `0` whenever `required`, which is not the same as no entry and should satisfy "required" for an integer

Reply:
Yes, good point. This went from complex to simple and back to complex for me. I just realized late last night that this creating more issues in the core than I first expected after my initial commits. I will go over this once again and take care of the `0` situation and also try to address negative max.